neural_network/activation_functions/squareplus.py

```diff
@@ -0,0 +1,38 @@
+"""
+Squareplus Activation Function
+
+Use Case: Squareplus designed to enhance positive values and suppress negative values.
+For more detailed information, you can refer to the following link:
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Squareplus
+"""
+
+import numpy as np
+
+
+def squareplus(vector: np.ndarray, beta: float) -> np.ndarray:
+    """
+    Implements the SquarePlus activation function.
+
+    Parameters:
+        vector (np.ndarray): The input array for the SquarePlus activation.
+        beta (float): size of the curved region
+
+    Returns:
+        np.ndarray: The input array after applying the SquarePlus activation.
+
+    Formula: f(x) = ( x + (x^2 + b)**0.5 ) / 2
```

Comment:
```suggestion
    Formula: f(x) = ( x + sqrt(x^2 + b) ) / 2
```
Small fix in the formula to make it more readable